src/components/posts/CollegeAdmissions/essay/EssayContainer.js

```diff
@@ -0,0 +1,128 @@
+import React, {useEffect, useReducer, useState} from "react";
+import WrittenEssay from "./WrittenEssay";
+import ScoreBar from "./ScoreBar";
+import WordSuggestionsBar from "./WordSuggestionsBar";
+import "./essay.css";
+
+//words reducer set-up
+
+//our wordSections
+//changeable sections will also have a displayed property
+const wordSections = [
+    {
+        plain: "This is our example"
+    },
+    {
+        changeable: ["essay", "activity", "game"],
+        scores: [10,20,30]
+    },
+    {
+        plain: "made by our devs! What's your favorite"
+    },
+    {
+        changeable: ["meal", "food", "dish"],
+        scores: [10,20,30]
+    },
+    {
+        plain: "to cook?"
```

Comment:
is there a reason why the "to cook?" is separated from the "I know that I love"? Earlier we combined "made by our devs!" and "What's your favorite" together